packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx

```diff
@@ -516,7 +517,15 @@ export class AtomicSearchBox {
 
   private getAndFilterLeftSuggestionElements() {
     const suggestionElements = this.getSuggestionElements(this.leftSuggestions);
-    return getUniqueItemsByProperties(suggestionElements, ['query']);
+    // We want to remove duplicates elements for the same `query` property, but only when it's not empty.
+    // Since the custom search box suggestion system does not enforce the query property (it's optional),
+    // it's a system that custom search box suggestions component will use to provide a "title" or separator section.
+    // We don't want to detect multiple separator section with no query as duplicate
```

Comment:
Maybe an e2e test that covers the case would be better than the comment (which can be detailed in the test description)

Reply:
Yeah, I thought about adding a test. The problem here though is that it requires creating a "custom component" if you wish, or at least mimic the custom search box provider system. 

We cannot test this with standard atomic component.

I can try to add something that would not needlessly complexify the test setup...